src/main.rs

```diff
@@ -31,7 +32,7 @@ const TICK_RATE_MS: u64 = 100;
 #[clap(version, about, long_about= None)]
 pub struct Cli {
     /// number of words to use in test
-    #[clap(short = 'w', long, default_value_t = 15)]
+    #[clap(short = 'w', long, default_value_t = 2000)]
```

Comment:
Let's revert this to 15 before we merge. In fact if you want to add 
`const DEFAULT_NUM_WORDS: u64 = 15;` under the `TICK_RATE_MS` const above that would be great!